test/SourceKit/CodeComplete/complete_typerelation.swift.convertible.response

```diff
@@ -26,39 +26,39 @@
     },
     {
       key.kind: source.lang.swift.decl.function.method.instance,
-      key.name: "intanceReturnMyEnum(:)",
-      key.sourcetext: "intanceReturnMyEnum(<#T##self: MyEnum##MyEnum#>)",
-      key.description: "intanceReturnMyEnum(self: MyEnum)",
+      key.name: "instanceReturnMyEnum(:)",
+      key.sourcetext: "instanceReturnMyEnum(<#T##self: MyEnum##MyEnum#>)",
+      key.description: "instanceReturnMyEnum(self: MyEnum)",
       key.typename: "() -> MyEnum",
       key.context: source.codecompletion.context.thisclass,
       key.typerelation: source.codecompletion.typerelation.unrelated,
       key.num_bytes_to_erase: 0,
-      key.associated_usrs: "s:21complete_typerelation6MyEnumO013intanceReturncD0ACyF",
+      key.associated_usrs: "s:21complete_typerelation6MyEnumO013instanceReturncD0ACyF",
       key.modulename: "complete_typerelation"
     },
     {
       key.kind: source.lang.swift.decl.function.method.instance,
-      key.name: "intanceReturnVoid(:)",
-      key.sourcetext: "intanceReturnVoid(<#T##self: MyEnum##MyEnum#>)",
-      key.description: "intanceReturnVoid(self: MyEnum)",
+      key.name: "instanceReturnVoid(:)",
+      key.sourcetext: "instanceReturnVoid(<#T##self: MyEnum##MyEnum#>)",
+      key.description: "instanceReturnVoid(self: MyEnum)",
       key.typename: "() -> Void",
       key.context: source.codecompletion.context.thisclass,
       key.typerelation: source.codecompletion.typerelation.invalid,
       key.num_bytes_to_erase: 0,
-      key.associated_usrs: "s:21complete_typerelation6MyEnumO17intanceReturnVoidyyF",
+      key.associated_usrs: "s:21complete_typerelation6MyEnumO17instanceReturnVoidyyF",
       key.modulename: "complete_typerelation"
     },
     {
       key.kind: source.lang.swift.decl.function.method.instance,
-      key.name: "intanceReturnVoidDeprecated(:)",
-      key.sourcetext: "intanceReturnVoidDeprecated(<#T##self: MyEnum##MyEnum#>)",
-      key.description: "intanceReturnVoidDeprecated(self: MyEnum)",
+      key.name: "instanceReturnVoidDeprecated(:)",
+      key.sourcetext: "instanceReturnVoidDeprecated(<#T##self: MyEnum##MyEnum#>)",
+      key.description: "instanceReturnVoidDeprecated(self: MyEnum)",
       key.typename: "() -> Void",
       key.context: source.codecompletion.context.thisclass,
       key.typerelation: source.codecompletion.typerelation.invalid,
       key.num_bytes_to_erase: 0,
       key.not_recommended: 1,
-      key.associated_usrs: "s:21complete_typerelation6MyEnumO27intanceReturnVoidDeprecatedyyF",
+      key.associated_usrs: "s:21complete_typerelation6MyEnumO27instanceReturnVoidDeprecatedyyF",
```

Comment:
This likely requires updating beyond direct substitution of the misspelled terms with corrected terms since it involves a mangled name.

Reply:
Might be simplest to drop the `instance` spelling fixes in the interest of getting the rest of the changes in.